lib/modules/search.js

```diff
@@ -300,32 +300,37 @@ function drawSearchResults(query, results) {
 	} else {
 		$resultsContainer.find('#SearchRES-results-hidden').removeClass('advancedResults');
 	}
+
 	if (!results || !results.length) {
 		$resultsContainer.find('#SearchRES-results').hide();
 	} else {
-		const resultsList = $('#SearchRES-results', $resultsContainer).show();
+		$resultsContainer.find('#SearchRES-results').show();
+
+		const resultsList = document.getElementById('SearchRES-results');
 
-		resultsList.empty();
+		resultsList.innerHTML = '';
 		for (const result of results) {
 			const element = drawSearchResultItem(result);
-			resultsList.append(element);
+			resultsList.appendChild(element);
 		}
 	}
 }
 
 function drawSearchResultItem(result) {
-	const element = $('<li>')
-		.addClass('SearchRES-result-item')
-		.toggleClass('advanced', !!result.advanced)
-		.data('SearchRES-result', result);
-
-	$('<span>', { class: 'SearchRES-result-copybutton res-icon' })
-		.html('&#xF159')
-		.appendTo(element)
-		.attr('title', 'copy this for a comment');
+	const element = document.createElement('li');
+	element.classList.add('SearchRES-result-item');
+	if (result.advanced) {
+		element.classList.add('advanced');
+	}
 
 	const details = searchResultTemplate(result);
-	element.append(details);
+	element.innerHTML = details;
+
+	const copybutton = document.createElement('span');
```

Comment:
You can also use the helper function `CreateElement.icon('F159', 'span', 'SearchRES-result-copybutton', i18n('searchCopyResultForComment'));`

and in `en.json` add `searchCopyResultForComment`